cmd/docgenerate/main.go

```diff
@@ -136,6 +141,27 @@ func makeTempRepo(srcdir string) string {
 	return d
 }
 
+func makeIndexInputFiles(wg *sync.WaitGroup, version, srcdir string) {
+	defer wg.Done()
+
+	jsPath := path.Join(srcdir, "docs", version, "searchIndex.js")
+	os.Create(jsPath)
```

Comment:
Even though errors are unlikely here, it's a good idea for us to trap potential errors and `panic` or `log.Fatal` on them. So there are several locations in this function where that might be a good idea:

- `os.Create`
- `f.WriteString`